package.json

```diff
@@ -36,15 +37,15 @@
     "eslint": "~8.56.0",
     "eslint-plugin-header": "~3.1.1",
     "shx": "~0.3.4",
```

Comment:
Suggestion: We should also update `shx` to `0.4.0`. This version is using an outdated version of `glob`.

Reply:
Yes, that sounds like a good idea and I will drop eslint as we discussed offline.

Reply:
Done ⬇️